.idea/vcs.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="IssueNavigationConfiguration">
+    <option name="links">
+      <list>
+        <IssueNavigationLink>
+          <option name="issueRegexp" value="#(\d+)" />
+          <option name="linkRegexp" value="https://github.com/Kotlin/kotlinx-io/issues/$1" />
+        </IssueNavigationLink>
```

Comment:
```suggestion
        <IssueNavigationLink>
          <option name="issueRegexp" value="#(\d+)" />
          <option name="linkRegexp" value="https://github.com/Kotlin/kotlinx-io/issues/$1" />
        </IssueNavigationLink>
        <IssueNavigationLink>
          <option name="issueRegexp" value="KT-(\d+)" />
          <option name="linkRegexp" value="https://youtrack.jetbrains.com/issues/KT-$1" />
        </IssueNavigationLink>
```

While we're here, I guess it might be worth to also highlight links to YT issues.

Reply:
Yeah, why not, good idea!